src/components/shared/WithSize.js

```diff
@@ -64,6 +85,8 @@ export function withSize<
       }
       const { width, height } = container.getBoundingClientRect();
       this.setState({ width, height });
+      const style = 'color: green; font-weight: bold;';
+      console.log(`[updateWidth]  %c"${width}, ${height}"`, style);
```

Comment:
This `console.log` probably should be removed. It's helpful for debugging this specific issue, but probably not useful to have in the code all the time.